src/components/accounts/two_factor/TwoFactorVerifyModal.js

```diff
@@ -58,16 +58,19 @@ const TwoFactorVerifyModal = ({ open, onClose }) => {
             await dispatch(resendTwoFactor())
         } catch(e) {
             setResendCode()
-            Mixpanel.track("2FA Resend errors", {error: e})
+            Mixpanel.track("2FA Modal Resend errors", {error: e})
             throw e
         } finally {
             setResendCode('resent')
-            Mixpanel.track("2FA Resend code")
+            Mixpanel.track("2FA Modal Resend code")
```

Comment:
I think `Mixpanel.track("2FA Modal Resend code")` should be placed at the end of `try`. `finally` will always happen regardless of success or failure